### PR TITLE
feat(HIVE-42): support forumpromo and eventpromo's from different brands

### PR DESCRIPTION
**Pull request by maxcbc**
# feat(HIVE-42): support forumpromo and eventpromo's from different brands

## What
- bumps n-eventpromo to v7.0.2
- adds support for eventpromos from different brands (i.e. passes the 'brand' prop to the component)
- adds support for eventpromos without a viewlink
- adds support for rendering 'forumpromo' if the api returns one
- updates demo with examples of the different brands and the 'forumpromo'

## Notes for reviewers
- The first commit in this PR contains all the changes to the actual 'application code', while the second commit only contains changes to the `localhost demo` contained within this repo.
- The newly added features in the PR will not be 'active' until next-magnet-api starts to return different event brands and forumpromos.
- The demo can be run locally with `make demo` and then navigating to http://localhost:5005

### CODE REVIEW

**Review by mymoon1020 — APPROVED** [review]

**Review by AliceArmstrong — CHANGES_REQUESTED** [review]
Nice work 🙌 One tiny formatting request, but otherwise looks good to go.

File: src/components/forumpromo/main.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/** @jsx h */
+import {Forumpromo} from '@financial-times/n-eventpromo';
+import {h, render} from '@financial-times/x-engine';
+import {dispatchTrackingEvent} from '../../lib/tracking';
+
+
```
[the remark this answers is not in the document]
Comment (maxcbc) [reply src/components/forumpromo/main.js]: Fixed 🙂

**Review by AliceArmstrong — APPROVED** [review]
🌟

**Comment by maxcbc** [comment]
@AliceArmstrong Thanks for the review & approve. Please can you merge it for me. I do not have the permissions to do so.

**Pull request merged**